gravitee-alert-engine-connector-ws/src/main/java/io/gravitee/ae/connector/ws/configuration/ConnectorConfiguration.java

```diff
@@ -176,6 +180,36 @@ public class ConnectorConfiguration {
     @Value("${alerts.alert-engine.ws.sendEventsOnHttp:false}")
     private boolean sendEventsOnHttp;
 
+    @Value("${alerts.alert-engine.ws.proxy.type:HTTP}")
+    private String proxyType;
+
+    @Value("${alerts.alert-engine.ws.useSystemProxy:false}")
+    private boolean useSystemProxy;
+
+    @Value("${alerts.alert-engine.ws.proxy.http.host:#{systemProperties['http.proxyHost'] ?: 'localhost'}}")
+    private String proxyHttpHost;
+
+    @Value("${alerts.alert-engine.ws.proxy.http.port:#{systemProperties['http.proxyPort'] ?: 3128}}")
+    private int proxyHttpPort;
+
+    @Value("${alerts.alert-engine.ws.proxy.http.username:#{null}}")
+    private String proxyHttpUsername;
+
+    @Value("${alerts.alert-engine.ws.proxy.http.password:#{null}}")
+    private String proxyHttpPassword;
+
+    @Value("${alerts.alert-engine.ws.proxy.https.host:#{systemProperties['https.proxyHost'] ?: 'localhost'}}")
+    private String proxyHttpsHost;
+
+    @Value("${alerts.alert-engine.ws.proxy.https.port:#{systemProperties['https.proxyPort'] ?: 3128}}")
+    private int proxyHttpsPort;
+
+    @Value("${alerts.alert-engine.ws.proxy.https.username:#{null}}")
+    private String proxyHttpsUsername;
+
+    @Value("${alerts.alert-engine.ws.proxy.https.password:#{null}}")
+    private String proxyHttpsPassword;
+
```

Comment:
I have my set of question here:

I know for instance that in the WebhookNotifier we're using the prefix `httpClient.proxy`

https://github.com/gravitee-io/gravitee-notifier-webhook/blob/47d6b0ee5bbaf62c042651e9d40d75a89a25aa07/src/main/java/io/gravitee/notifier/webhook/WebhookNotifier.java#L50

Should we isolate it for Alert engine connector ? Or should we prefix it the same way ?